src/core/agent/services/identifierService.ts

```diff
@@ -139,20 +161,57 @@ class IdentifierService extends AgentService {
     };
   }
 
+  async resolvePendingIdentifier() {
```

Comment:
Not sure what this function is doing. We will always add the end role in `keriaNotificationService` now. What you are meant to do on start-up is get the basic record, and if there are any names in the basic record, try to create them.

This in case we add them to the basic record but crash before the event callback completes. So on next start-up we cover it.